pyproject.toml

```
@@ -0,0 +1,6 @@
+[build-system]
+requires = [
+    "setuptools>=61",
```

```suggestion
    "setuptools>=62",
```

for this, but fine as is for me: https://github.com/pypa/setuptools/pull/3151